packages/databyss-editor/state/reducer.ts

```diff
@@ -406,8 +430,20 @@ export default (
       return draft
     }
   )
+
+  /*
+historyActions need to bypass the EditorHistory history stack
+  */
+
   if (onChange) {
-    onChange({ previousState: state, nextState, patches, inversePatches })
+    onChange({ 
+      previousState: state, 
+      nextState, 
+      patches, 
+      inversePatches,
+      undoAction: action.type === UNDO, 
```

Comment:
Maybe make this more explicit, like `skipHistory: bool` and it would be true if the type is UNDO or REDO, right?